### PR TITLE
Feature/iex setup

### PR DESCRIPTION
**Pull request by Zhi0105**
## Summary
 - Creating market model
 - Creating market controller
 - Updating rails credentials using API secret and publishable token
 - Configuring IEX API to connect on the project
 - Rendering fetched data coming from API
 
## Screenshot
Capture

## References
- https://github.com/dblock/iex-ruby-client
- https://iexcloud.io/docs/api/
- https://stackoverflow.com/questions/57389966/configuring-ruby-on-rails-application-with-iex-ruby-client-gem
- https://github.com/mumble-07/emporia-wallet-app/tree/master/app/services

### CODE REVIEW

**Review by kitkatpussycat — APPROVED** [review]

File: app/models/market.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  #     file_data = file.readlines.map(&:chomp)
+
+  #     file_data.each do |data|
+  #         Market.create(
+  #             name: client.company(data),
+  #             market_symbol: data,
+  #             curr_price: client.price(data)
+  #         )
+  #     end
+  # end
```
Comment (analyn-cajocson) [inline app/models/market.rb]: removed the unnecessary comments

File: app/views/markets/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    <tbody>
+        <% @markets.each do |market| %>
+        <tr>
+            <td><%= market.market_symbol %></td>    
+            <td><%= market.name %></td>    
+            <td><%= market.curr_price %></td>    
+        </tr>
+        <% end %>
+    </tbody>
+</table>
```
Comment (analyn-cajocson) [inline app/views/markets/index.html.erb]: add new line

File: db/seeds.rb [hunk 3]
```diff
@@ -5,4 +5,18 @@
...
+    endpoint: 'https://sandbox.iexapis.com/v1'
+)
+
+    Market.destroy_all
+    file = File.open('app/api/stock_lists/market_symbol.txt')
+    file_data = file.readlines.map(&:chomp)
+    
+    file_data.each do |data|
+        Market.create( name: client.company(data).company_name, market_symbol: data, curr_price: client.price(data))
+    end
```
Comment (analyn-cajocson) [inline db/seeds.rb]: add new line here

**Review by analyn-cajocson — CHANGES_REQUESTED** [review]

File: app/models/market.rb (hunk 1 above)
Comment (Zhi0105) [reply app/models/market.rb]: > removed the unnecessary comments

done.

File: app/views/markets/index.html.erb (hunk 2 above)
Comment (Zhi0105) [reply app/views/markets/index.html.erb]: > add new line

done

File: db/seeds.rb (hunk 3 above)
Comment (Zhi0105) [reply db/seeds.rb]: > add new line here

done.

Comment (analyn-cajocson) [inline db/seeds.rb]: can you fix the code alignment also?

**Review by analyn-cajocson — CHANGES_REQUESTED** [review]

File: db/seeds.rb (hunk 3 above)
Comment (Zhi0105) [reply db/seeds.rb]: > can you fix the code alignment also?

done

**Review by analyn-cajocson — APPROVED** [review]

**Pull request merged**